datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -126,6 +138,23 @@ def start(ctx, check, env, agent, python, dev, base, env_vars):
     for key, value in metadata.get('env_vars', {}).items():
         env_vars.setdefault(key, value)
 
+    if profile_memory:
+        env_vars['DD_TRACEMALLOC_DEBUG'] = '1'
+        env_vars['DD_TRACEMALLOC_WHITELIST'] = check
+
+        instances = config
+        if isinstance(config, dict):
+            instances = config.get('instances', [config])
+
+        for instance in instances:
+            instance['__memory_profiling_tags'] = ['env:{}'.format(env)]
+
+            if on_ci:
+                instance['__memory_profiling_tags'].append('branch:{}'.format(get_current_branch()))
+                instance['min_collection_interval'] = metadata.get(
+                    'sampling_collection_interval', DEFAULT_SAMPLING_COLLECTION_INTERVAL
+                )
```

Comment:
`min_collection_interval` this might impact some e2e tests, it might worth triggering all tests to check that.

Reply:
Nevermind, all tests are already running.